server/svix-server/src/core/cache/redis.rs

```diff
@@ -112,12 +116,23 @@ mod tests {
         dotenv::dotenv().ok();
         let cfg = crate::cfg::load().unwrap();
 
-        let redis_pool = bb8::Pool::builder()
-            .build(RedisConnectionManager::new(cfg.redis_dsn.as_deref().unwrap()).unwrap())
-            .await
-            .unwrap();
-
-        let cache = new(redis_pool.clone());
+        let redis_cache = match cfg.cache_type {
+            CacheType::RedisCluster => {
+                let mgr =
+                    crate::redis::create_redis_pool(cfg.redis_dsn.as_ref().unwrap().as_str(), true)
+                        .await;
+                Some(mgr)
+            }
+            _ => {
+                let mgr = crate::redis::create_redis_pool(
+                    cfg.redis_dsn.as_ref().unwrap().as_str(),
+                    false,
+                )
+                .await;
```

Comment:
Using true/false for this is very confusing. I actually missed it the first two times around. :|

Reply:
The reason it's true/false is b/c there are 2 different config enums, `CacheType` and `QueueType` for which this is applicable and wanted to easily reuse this.

Reply:
Yeah, I understand that, though having two functions with different names (even if they call the same private function internally that uses true/false) makes things much clearer. Here I easily missed it.